Sources/RealDeviceMap/Modell/Pokemon.swift

```diff
@@ -487,10 +487,13 @@ class Pokemon: JSONConvertibleObject, WebHookEvent, Equatable, CustomStringConve
                 "pokemon": ranking.pokemon.pokemon.rawValue,
                 "form": ranking.pokemon.form?.rawValue ?? 0,
                 "gender": ranking.pokemon.gender?.rawValue ?? 0,
-                "rank": ranking.response?.rank as Any,
+                "rank": ranking.response?.denseRank as Any,
```

Comment:
shouldn't there be a config variable in yml as to which type of "rank" instead of defaulting to dense?

Reply:
In my mind, no. This maintains backwards compatibility and software can choose which rank type to use from the new variables (or whatever you like to call them).